### PR TITLE
Build changes for iOS v2.9.2 release

### PR DESCRIPTION
**Pull request by mcbouslog**

### CODE REVIEW

File: ios/ZooniverseMobile/Info.plist [hunk 1]
```diff
@@ -17,7 +17,7 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>$(MARKETING_VERSION)</string>
+	<string>2.9.2</string>
```
Comment (lcjohnso) [inline ios/ZooniverseMobile/Info.plist:20]: Was it necessary to eliminate this variable, or was this your preference?  Personally, I like setting the value directly in Info.plist, but then can we also remove the two commands to set the variable in `project.pbxproj`?

Also: we should change the iOS build instructions [GDoc link](https://docs.google.com/document/d/1kMbryj3tvJhnXkdIgHcnmOJQqwh_VMLspuCFYJfn_0s/edit) to match.

**Review by lcjohnso — APPROVED** [review]
Good to go, but one comment to future-proof iOS deploys and avoid confusion.

File: ios/ZooniverseMobile/Info.plist (hunk 1 above)
Comment (mcbouslog) [reply ios/ZooniverseMobile/Info.plist:20]: I was able to delete the references to `MARKETING_VERSION` from project.pbxproj and ran the app locally and archived/built the app for release without any noticeable issue (though I did not upload archive to App Store).

Comment (mcbouslog) [reply ios/ZooniverseMobile/Info.plist:20]: I also updated the related GDoc iOS - App Store, removing reference to the MARKETING_VERSION in the project.pbxproj file.

**Pull request merged**